.gitignore

```diff
@@ -1,4 +1,4 @@
-pep-0000.txt
+pep-0000.rst

```

Comment:
Old checkouts may still have the text file around, so best to keep both files here for now.